src/receiver.h

```diff
@@ -594,7 +594,8 @@ string getStepText(const string& dataDirectory, const string& fileName, int heig
 		{
 			cout << "Downloading " << stepFileName << " base file." << directorySubName << endl;
 			string peerText = string("_beginpeers\n");
-			peerText += string("https://raw.github.com/sidhujag/devcoin/master/src/node2/testnet3/receiverTestNet/receiverTestNet.csv\n");
+			peerText += string("http://node1.devcoin.cloud/testnet_receiver_files/receiverTestNet.csv\n");
+			peerText += string("http://devcoinresource.com/testnet_receiver_files/receiverTestNet.csv\n");
```

Comment:
File not found

Reply:
That is correct... you won't find the file by browsing it.  The same is true for the production files.
The number of receiver is appended to this string to find the file name. 
If you compile the source, the wallet finds and downloads the receiver correctly.